### PR TITLE
Adds blinding + blindfolds

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
Works on my machine.

**Screenshots**

https://user-images.githubusercontent.com/60792108/172264001-4b26af30-aead-4576-9f4f-caf661129151.mp4

**Changelog**

:cl: Rane
- add: Added blindfolds!
- add: Added a rare, funny salvage item.

### CODE REVIEW

**Comment by Emisse** [comment]
finally some good erp content

File: Content.Client/Entry/EntryPoint.cs [hunk 1]
```diff
@@ -191,6 +192,7 @@ public override void PostInit()
             overlayMgr.AddOverlay(new CircleMaskOverlay());
             overlayMgr.AddOverlay(new FlashOverlay());
             overlayMgr.AddOverlay(new RadiationPulseOverlay());
+            overlayMgr.AddOverlay(new BlindOverlay());
```
Comment (metalgearsloth) [inline Content.Client/Entry/EntryPoint.cs:195]: This should be added or removed dynamically.

Comment (Elijahrane) [reply Content.Client/Entry/EntryPoint.cs:195]: How would you do that and why do none of the other overlays do that?

**Review by wrexbe — CHANGES_REQUESTED** [review]

**Comment by wrexbe** [comment]
What if after the places you change  `blindComp.Sources`
You raise a BlindedStateChangedEvent (or some other name)
Then on the BlindingSystem you subscribe to that event, and turn the light manager on/off, and add/remove the overlay
Then the overlay doesn't need to do any logic, and you can make all the blinding system subscriptions call an "UpdateBlindEffect" method.

**Comment by Elijahrane** [comment]
> Then on the BlindingSystem you subscribe to that event, and turn the light manager on/off, and add/remove the overlay
The problem is that you need to change light manager one frame before the overlay. Doing this in the same subscription would preserve the bug

**Comment by wrexbe** [comment]
What do you know about this bug? Is it something that should be fixed, or a limitation?

**Comment by Elijahrane** [comment]
> What do you know about this bug? Is it something that should be fixed, or a limitation?

Toggling light manager on/off takes place the next frame. During that frame you can see through walls.

**Review by wrexbe — APPROVED** [review]

**Comment by wrexbe** [comment]
The shader you were using does something different now on master, so I switched it to the other circle mask shader

**Pull request merged**